Dockerfile

```diff
@@ -1,21 +1,19 @@
-FROM swift:4.0
-
-ENV SKIP_TESTS 1
-
-RUN swift --version
+FROM norionomura/swift:41
```

Comment:
[This image](https://hub.docker.com/r/norionomura/swift/) has been updated pretty regularly and seems reliable.